index.html

```diff
@@ -42,5 +42,177 @@ <h1 class="name"> Hey there <br>
           <img src="images/Disabled.svg" alt="drop-down">
         </div>
     </section>
+    <section class="line-break">
+            <h2 class="works">My Recent <br> Works</h2>
+    </section>
+    
+      <section class="work-section">
+        <div  class="pictureholder-container">
+            <img src="Images 2/placeholder.svg" alt="placeholder" class="placeholder">
+            <div class="text-container">
+              <h5  class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+              <ul class="text">
+                <li class="text-ruby">Ruby on Rails</li>
+                <li class="text-css">CSS</li>
+                <li class="text-java">JavaScript</li>
+                <li class="text-html">HTML</li>
+              </ul>
+              <div class="button1"><button class="button-text" type="button">See project </button></div>
+            </div>
+        </div>
```

Comment:
Great job on implementing the cards! However, there are a few areas that require improvement to align with the design and improve the overall user experience.

- [ ]  [Optional] Firstly, consider replacing the `<div>` tag with an `<article>` tag for the card element. This will ensure proper semantic structure and accurately represent the purpose of the card within the page. :grinning:
- [ ] [Optional] Additionally, the appearance of the cards does not match the design. Pay attention to aligning the cards and their elements, including spacing, and border-radius, to closely resemble the design. Make the necessary adjustments to achieve a consistent and visually appealing result in the card. :grinning:

| Your | Professional |
| --- | --- |
| ![image](https://github.com/Wils-FOURR-speed/Portfolio-setup-and-mobile-first-project/assets/55840999/1ea39445-2192-402a-bc53-a4907f11e64e)  |  ![image](https://github.com/Wils-FOURR-speed/Portfolio-setup-and-mobile-first-project/assets/55840999/b54fa4bf-ae05-4de3-b7f9-2a4124f15dba)  |